### ISSUE TITLE
Every stream recorded is marked as vod deleted from twitch when no internet is available.

### ISSUE THREAD

**Issue by KurtzPT** [MrBrax/LiveStreamDVR#198]
Basically what titles says, my Internet failed for two hours and in that time LiveStreamDVR started the vods checks and instead of ignoring the result (because no internet), it marked every single recordings as deleted from vod on twitch:

![image](https://user-images.githubusercontent.com/14930057/168292283-8b9c374a-ac33-42cc-8f0b-585f34ced8a0.png)

**Comment by MrBrax** [comment]
got a bluescreen while re-enabling my network interface

very cursed issue

**Comment by MrBrax** [comment]
https://github.com/axios/axios/issues/2986

i guess it's a bit more complicated than just a message check? it did return no address or something for me, but there might be multiple errors

**Comment by KurtzPT** [comment]
I'm not sure how the vod check is done but shouldn't just mark something as deleted ONLY if it gets a response from twitch but the page does not have the video? That way it doesn't matter if twitch is down or if I have an internet connectivity issue, it should not mark the vods as deleted in both of those cases.

**Comment by KurtzPT** [comment]
Also how can I know fix this mess? Is there any way of requesting LiveStreamDVR to check every single record for vod availability ?

**Comment by MrBrax** [comment]
click the refresh button next to the vod id, i pushed an addition to have it on completely deleted ones too but can't remember if i merged it

**Comment by KurtzPT** [comment]
> click the refresh button next to the vod id, i pushed an addition to have it on completely deleted ones too but can't remember if i merged it

But that's one by one, no way of doing it all? it's still a bunch of records.

**Comment by KurtzPT** [comment]
![image](https://user-images.githubusercontent.com/14930057/168390130-11396a1f-3fb5-4a86-ae48-0b4d1a6e78fe.png)

Nevermind, there's no refresh button while its marked as deleted.

**Comment by MrBrax** [comment]
i think the cron should fix it, or going to /cron/check_deleted_vods

**Comment by KurtzPT** [comment]
> i think the cron should fix it, or going to /cron/check_deleted_vods

`https://<host>/cron/check_deleted_vods` ?

Doesn't seem to do anything.

**Comment by MrBrax** [comment]
right, /api/v0/cron/check_deleted_vods

**Comment by KurtzPT** [comment]
I remember having a page showing these cron, but not anymore it seems they disappeared. Anyway that worked. 👍

**Comment by MrBrax** [comment]
yeah because they run internally now, if enabled in the scheduler option in settings

**Comment by KurtzPT** [comment]
> yeah because they run internally now, if enabled in the scheduler option in settings

I have them enabled, maybe there's a way of having a button in the tools page to run these checks manually?

**Comment by KurtzPT** [comment]
I just noticed now last stream of Asmongold (Yesterday) is marked as VOD deleted and says "The VOD probably never got saved."

But that's not true, the VOD is here: https://www.twitch.tv/videos/1481888628

Possible new issue?

**Comment by MrBrax** [comment]
does he have the new thing that vods are not shown by default maybe?

**Comment by KurtzPT** [comment]
> does he have the new thing that vods are not shown by default maybe?

Not sure what is that, I found the vod by just checking last broadcasts on his page.

**Comment by MrBrax** [comment]
![1652556152404](https://user-images.githubusercontent.com/1517911/168445643-d1f4322e-97c6-4878-847f-c0b9da458e51.png)

so he's streaming right now but the vod isn't visible. not really anything i can do other than add a "match vod" cronjob i guess, for when the channel makes the vod public?

**Comment by KurtzPT** [comment]
I didn't know hiding vods while streaming was a thing. I guess the match vod cronjob would be a nice thing to have then.

**Comment by KurtzPT** [comment]
How frequently is this cron running? Trying to figure out if it's working because I have a streamer that all vods are marked as deleted when it shouldn't have.

**Comment by MrBrax** [comment]
> How frequently is this cron running? Trying to figure out if it's working because I have a streamer that all vods are marked as deleted when it shouldn't have.

it should be running twice a day and output to the log, however if the vod has an id it won't check

**Comment by KurtzPT** [comment]
Seems to be working the cron.

**Comment by KurtzPT** [comment]
@MrBrax not sure if this is related but I'm starting to see the following on some recordings:

![image](https://user-images.githubusercontent.com/14930057/171187628-ee4a0666-14dd-4d77-8907-43a4b6a8e080.png)

That red exclamation mark, never seen that before. What is it and how can I fix it (if there's something to fix?)
